### DISCUSSION TITLE
Welcome to cloud-native-kiosk Discussions!

### DISCUSSION THREAD

**Discussion by yuyatinnefeld** [yuyatinnefeld/cloud-native-kiosk#69] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by yuyatinnefeld** [comment]
change the backend service from Python to Golang or Rust?

**Reply by AndrewMC25** [reply]
Ey man, the documentation have a link not very appropriate, check that
